src/gtfs/mod.rs

```diff
@@ -261,13 +261,13 @@ fn read<H>(
     config_path: Option<impl AsRef<Path>>,
     prefix: Option<String>,
     on_demand_transport: bool,
+    on_demand_transport_comment: Option<String>,
```

Comment:
Would it be possible to regroup all this parameters into on structure to avoid getting more and more parameters to this function?

Maybe something like (not tested)

```rust
struct Configuration<P: AsRef<Path>> {
  config_file: P,
  prefix: Option<String>,
  on_demand_transport: bool,
  on_demand_transport_comment: Option<String>,
}
```

Reply:
clippy limit for Result (3 or 4 max), but not for inputs :thinking:

Reply:
For function, the limit for `clippy` seems to be 7
https://rust-lang.github.io/rust-clippy/master/index.html#too_many_arguments

For returned values, it's always been one (you can't return more). But I think you're talking about big tuples in `Result`? This must have been a different rule about type complexity
https://rust-lang.github.io/rust-clippy/master/index.html#type_complexity

Reply:
Limit is 3 for tuple in ```Result``` yes